src/js/messaging/actions/folders.js

```diff
@@ -1,5 +1,6 @@
 import { api } from '../config';
 import { createUrlWithQuery } from '../utils/helpers';
+import assign from 'lodash/fp/assign';
```

Comment:
Nitpick: Can these imports go at the top so that third-party library imports go first?